commit 11dadbc333c6480a0944c7ad3147d24b360bdb4a
Author: Andy Whitcroft <apw@canonical.com>
Date:   Thu Oct 1 18:01:29 2015 +0100

    UBUNTU: SAUCE: leds: lp55xx -- do not force use of the fallback loader
    
    The lp55xx driver uses the firmware loader to get its firmware.  Do not
    force us to use the fallback the kernel one should be sufficient.
    
    Signed-off-by: Andy Whitcroft <apw@canonical.com>

diff --git a/drivers/leds/Kconfig b/drivers/leds/Kconfig
index 52ea34e337cd..0ceb74fc6126 100644
--- a/drivers/leds/Kconfig
+++ b/drivers/leds/Kconfig
@@ -280,7 +280,6 @@ config LEDS_LP55XX_COMMON
 	tristate "Common Driver for TI/National LP5521/5523/55231/5562/8501"
 	depends on LEDS_LP5521 || LEDS_LP5523 || LEDS_LP5562 || LEDS_LP8501
 	select FW_LOADER
-	select FW_LOADER_USER_HELPER
 	help
 	  This option supports common operations for LP5521/5523/55231/5562/8501
 	  devices.
